spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/SparkTypeConverter.java

```diff
@@ -101,6 +131,32 @@ public static DataType toSparkType(Type gravitinoType) {
     } else if (gravitinoType instanceof Types.TimestampType
         && !((Types.TimestampType) gravitinoType).hasTimeZone()) {
       return DataTypes.TimestampNTZType;
+    } else if (gravitinoType instanceof Types.ListType) {
+      Types.ListType listType = (Types.ListType) gravitinoType;
+      return ArrayType.apply(toSparkType(listType.elementType()), listType.elementNullable());
```

Comment:
`DataTypes` provides helper method to create Array and Map, could you use helper methods?

Reply:
fixed.